pyrealm/splash/splash.py

```diff
@@ -144,14 +155,14 @@ def estimate_initial_soil_moisture(
             # Check the shape is the same as the shape of a slice along axis 0
             if wn_init.shape != self.tc[0].shape:
                 raise ValueError("Incorrect shape in wn_init")
-            wn_start = wn_init
+            wn_start = bounds_checker(wn_init, 0, self.kWm)
         else:
```

Comment:
I think there's a good case here for making this an Exception. 

`bounds_checker` only warns because remotely sensed data inputs often have wild values - that function used to raise Exceptions but it just happened all the time and made it unusable. But here, we need a more explicit test that  values are in `[0, 150]` and it should fail if they aren't.

Reply:
Fixed in the latest commit.